### DISCUSSION TITLE
Splitted account

### DISCUSSION THREAD

**Discussion by simoncerman** [community/community#22545] [New to GitHub]
<p>Hi!<br>
I have problem with commits. Im logged in VS Code by guthub, but my account was somehow splitted?<br>
You can see 2 simoncerman. But i have only one account. Commits are not showed on graph of activity.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/a/ba24849ca19d6c7bdcf879b4f12345813d2cb962.png" data-download-href="/uploads/short-url/qyH32jEeS6zzTGW3Trez1Ovwwee.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/77004989/181071750-7f5dd3f6-2b22-42e4-b8b4-cb69e1d95319.png" alt="image" data-base62-sha1="qyH32jEeS6zzTGW3Trez1Ovwwee" width="689" height="321" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/a/ba24849ca19d6c7bdcf879b4f12345813d2cb962_2_689x321.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/a/ba24849ca19d6c7bdcf879b4f12345813d2cb962_2_1033x481.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/b/a/ba24849ca19d6c7bdcf879b4f12345813d2cb962.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/a/ba24849ca19d6c7bdcf879b4f12345813d2cb962_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1374×640 52.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>Check the author email address in those commits, usually a mistake there is the reason for stuff like this. What’s odd is that the other commits list the same name, but not the “unknown user” grey octocat icon. :thinking:</p>

**Comment by simoncerman** [comment]
<p>Somehow i have splitted account in two mails but only one was in github settings. So easy solution was adding the secound email! It fix all. Old commits load to graph and now it is all from me.</p>
